### PR TITLE
JUnit 5 timebox

### PR DESCRIPTION
**Pull request by omnivion**
### Information
- JIRA story QPPCT-529.

### Checklist 
- [x] All JUnit tests pass (`mvn clean verify`).
- [X] New unit tests written to cover new functionality.
- [X] Added and updated JavaDocs for non-test classes and methods.
- [X] No local design debt. Do you feel that something is "ugly" after your changes?
- [X] Updated documentation (`README.md`, etc.) depending if the changes require it.

I targeted usages of org.junit.Assert this time.

### CODE REVIEW

File: converter/src/test/java/gov/cms/qpp/conversion/encode/AciProportionDenominatorEncoderTest.java [hunk 1]
```diff
@@ -1,24 +1,26 @@
...
 
-public class AciProportionDenominatorEncoderTest {
+class AciProportionDenominatorEncoderTest {
 
 	private Node aciProportionDenominatorNode;
 	private Node numeratorDenominatorValueNode;
 	private List<Node> nodes;
 	private JsonWrapper json;
 
 	@Before
```
Comment (halprin) [inline converter/src/test/java/gov/cms/qpp/conversion/encode/AciProportionDenominatorEncoderTest.java]: Needs to be updated to `@BeforeEach`.

File: converter/src/test/java/gov/cms/qpp/conversion/encode/AciProportionNumeratorEncoderTest.java [hunk 2]
```diff
@@ -1,24 +1,26 @@
...
 
-public class AciProportionNumeratorEncoderTest {
+class AciProportionNumeratorEncoderTest {
 
 	private Node aciProportionNumeratorNode;
 	private Node numeratorDenominatorValueNode;
 	private List<Node> nodes;
 	private JsonWrapper jsonWrapper;
 
 	@Before
```
Comment (halprin) [inline converter/src/test/java/gov/cms/qpp/conversion/encode/AciProportionNumeratorEncoderTest.java]: Needs to be updated to `@BeforeEach`.

File: converter/src/test/java/gov/cms/qpp/conversion/encode/AciSectionEncoderMultiMeasureTest.java [hunk 3]
```diff
@@ -45,7 +47,7 @@
 	private List<Node> nodes;
 
 	@Before
```
Comment (halprin) [inline converter/src/test/java/gov/cms/qpp/conversion/encode/AciSectionEncoderMultiMeasureTest.java]: Needs to be updated to `@BeforeEach`.

File: converter/src/test/java/gov/cms/qpp/conversion/validate/ClinicalDocumentValidatorTest.java [hunk 4]
```diff
@@ -23,17 +23,17 @@
 import gov.cms.qpp.conversion.model.error.TransformException;
 import gov.cms.qpp.conversion.model.error.correspondence.DetailsErrorEquals;
 
-public class ClinicalDocumentValidatorTest {
+class ClinicalDocumentValidatorTest {
 
 	private static final String CLINICAL_DOCUMENT_ERROR_FILE = "angerClinicalDocumentValidations.err.json";
 
 	@After
```
Comment (halprin) [inline converter/src/test/java/gov/cms/qpp/conversion/validate/ClinicalDocumentValidatorTest.java]: Needs to be updated to `@AfterEach`.

File: converter/.gitignore [hunk 5]
```diff
@@ -1,3 +1,4 @@
 DefaultRun.json
 DefaultRun.nodes
-src/main/java/gov/cms/qpp/conversion/correlation/model/*
\ No newline at end of file
+src/main/java/gov/cms/qpp/conversion/correlation/model/*
+/bin/
```
Comment (halprin) [inline converter/.gitignore]: Ignore `/bin/`?

Comment (omnivion) [reply converter/.gitignore]: This happened automatically. I can remove it, but I guess a bin folder popped up and got ignored

**Pull request merged**